.travis.yml

```diff
@@ -7,16 +7,16 @@ env:
     - IPFS_REUSEPORT=false
 
 go:
-    - 1.8
-    - 1.9
+    - 1.8.x
```

Comment:
Can probably remove this, we don't bother with go backwards compatibility. Too many shiny new features.